packages/driver/cypress/e2e/cypress/cy.cy.js

```diff
@@ -133,6 +133,16 @@ describe('driver/src/cypress/cy', () => {
         expect(userInvocationStack).to.include('.cy.js')
       })
     })
+
+    it('supports cy.state(\'subject\') for backwards compatability', () => {
+      cy.stub(Cypress.utils, 'warning')
+      const a = {}
+
+      cy.wrap(a).then(() => {
+        expect(cy.state('subject')).to.equal(a)
```

Comment:
nit: I prefer more descriptive longer variable names.

```suggestion
      const subject = {}

      cy.wrap(subject).then(() => {
        expect(cy.state('subject')).to.equal(subject)
```